docs/synapse/userguides/storm_ref_cmd.rstorm

```diff
@@ -2582,8 +2707,30 @@ node's node identifier (node ID / iden) so that only the first node with a given
   resolved to:
 
 .. storm-pre:: [inet:dns:a=(gdforum.info, 111.90.148.124) inet:dns:a=(live-settings.com, 209.99.40.222) inet:dns:a=(drive-google.ga, 141.8.224.221) ] -> inet:fqdn [+#rep.threatconnect.fancybear ]
-.. storm-cli:: inet:fqdn#rep.threatconnect.fancybear -> inet:dns:a -> inet:ipv4 | uniq
+.. storm-pre:: inet:fqdn#rep.threatconnect.fancybear -> inet:dns:a -> inet:ipv4 | uniq
+::
+  
+  inet:fqdn#rep.threatconnect.fancybear -> inet:dns:a -> inet:ipv4 | uniq
+
+
+- Lift a set of network flow (``inet:flow``) nodes and unique (de-duplicate) them based on the source IPv4 address:
+
+.. storm-pre:: [ ( inet:flow=* :src:ipv4=1.1.1.1 :dst:ipv4=2.2.2.2 ) ( inet:flow=* :src:ipv4=11.11.11.11 :dst:ipv4=3.3.3.3 ) ( inet:flow=* :src:ipv4=1.1.1.1 :dst:ipv4=4.4.4.4 ) ( inet:flow=* :src:ipv4=1.1.1.1 :dst:ipv4=4.4.4.4 ) ( inet:flow=* :src:ipv4=121.121.121.121 :dst:ipv4=2.2.2.2 ) ]
+.. storm-pre:: inet:flow | uniq :src:ipv4
+::
+  
+  inet:flow | uniq :src:ipv4
+
+
+- Lift a set of network flow (``inet:flow``) nodes and de-duplicate them based on each unique combination of source and destination IPv4 addresses:
+
+.. storm-pre:: inet:flow | uniq (:src:ipv4, :dst:ipv4)
+::
+  
+  inet:flow | uniq (:src:ipv4, :dst:ipv4)
```

Comment:
Do we want to add an example of using a per-node variable to unique nodes?

Reply:
Added...it's a bit of a klugey example, but it's an example. Open to suggestions.

Reply:
Here is an example that is more "sample" driven:

```
- Nodes can be uniqued based on variables. Alert (``risk:alert``) nodes can be categorized in various ways. This
  includes ``:priority`` and ``:severity`` properties, both of which use a set of fixed text values (e.g., "low" vs.
  "highest") that  correspond to integers (e.g., 20 vs. 50). These integer values could be joined together in a
  variable to provide a sample of alerts which have unique combinations of those values:

.. storm-pre:: [ (risk:alert=* :name='Cat sitting in front of monitor' :priority=20 :severity=20) (risk:alert=* :name='Out of coffee' :priority=10 :severity=50) (risk:alert=* :name='Laptop BIOS update bricked system' :priority=50 :severity=50) (risk:alert=* :name='New version of favorite Power-Up released' :priority=50 :severity=50) ]
.. storm-pre:: risk:alert:priority +:severity $pri=:priority $sev=:severity $value=($pri, $sev) | uniq $value
::
  
  risk:alert:priority +:severity $pri=:priority $sev=:severity $value=($pri, $sev) | uniq $value
```

Reply:
updated - thanks!